salesagility/SuiteCRM
# Adding High Contrast Theme

## Description




The noon theme has been refactored into its current state with slight changes made to the other 4 themes to make way for the High Contrast Theme. 

## Motivation and Context

The High Contrast change brings a new theme to SuiteCRM. 
Documentation will be added for this theme too.
## How To Test This

1. Go to profile -> Layout Options 
2. Then select the "Noon Theme" -> Save
3. Now test functionality for all 5 themes and make sure dropdowns, edit views, detail views, etc look the way they are supposed to.
## Types of changes


### Final checklist
